<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_timeReport</class>
 <widget class="QDialog" name="Dialog_timeReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберите диапазон дат</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Добавить &gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="1" column="2" rowspan="3">
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>По всем проектам</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>&lt;&lt; Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Список отобранных проектов</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Полный список проектов</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Наименование проекта</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата последней задачи</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Выполненных задач</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Потрачено времени </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Потрачено денег</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>От</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="currentSection">
          <enum>QDateTimeEdit::DaySection</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>По всем датам</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="11" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Поиск проекта по подстроке</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="8" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>0 рублей 00 копеек</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Всего потрачено времени по всем проектам:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Всего потрачено денег по всем проектам:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Вывести отчёт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Экспорт</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
